### DISCUSSION TITLE
Who's Hiring (April 2024)

### DISCUSSION THREAD

**Discussion by manovotny** [vercel/next.js#64033] [Show and tell]
# What is this?

This discussion is for companies looking for people with Next.js experience. Both permanent positions and contract work are welcome.

## Companies

Please use the following format followed by a brief description of the role:

> COMPANY_NAME | ROLE | LOCATION | Full-time | $100-150k base | LINK

Please only post if you personally are part of the hiring company—no recruiting firms or job boards. Only one post per company.

Good luck with recruiting!

### Sample Job Post

Here's a good example of a job post.

> Vercel | Software Engineer | Remote (US, Canada, Europe, or Asia) | Full-time | $100-225k base | [Link](https://vercel.com/careers#positions)
>
> Vercel’s hiring across the board on various engineering teams and positions, but the above link is for our software engineering position. Vercel is innovating on and for the web and at the edge. We build tools and workflows developers love and deliver outstanding performance for end users.
>
> I've been at the company for 2 years and still find plenty of opportunity for impact, and love the teammates I'm working with. Feel free to apply using the link above or reach out to me directly.

---

If you hire someone through this job board, please [email me](mailto:michael.novotny@vercel.com) and let me know so I can send you some swag. 👍 

If you're a developer looking to get hired, please use #64034 instead.

Several companies have [hired someone](https://twitter.com/nutlope/status/1610667757565083648) as a result!

**Comment by hahnbeelee** [comment]
Mintlify | Full Stack Engineer | In-Person San Francisco | Full-time | $100k-$200k base | [Link](https://www.ycombinator.com/companies/mintlify/jobs/pk3AAR7-full-stack-engineer)

[Mintlify](https://mintlify.com/) is hiring a full-stack engineer. We utilize Next.js and Vercel to power thousands of company's documentation. If a company wants to create guides, tutorials, or API references, Mintlify serves as a toolkit to help them write, maintain, customize the look and feel, and host the documentation.

I'm one of the co-founders and the team consists of intrinsically motivated individuals. We like to have fun and work hard. If you love Next.js and innovating as much as I do, then please apply using the link!

**Reply by seyedahmaddev** [reply]
hi, do you have remote position?

**Reply by programhei** [reply]
hi, do you have remote position?

**Comment by nick-inkeep** [comment]
Inkeep | Full Stack Engineer | In-Person New York or SF | Full-time | 100k-160k base | [Link](https://join.inkeep.com/founding-engineer)

[Inkeep](https://inkeep.com) is hiring for full-stack JS engineer. We help devtool companies like Mintlify(^), Bun, Neon, Resend and others add AI search and chat to their documentation. Scope of the role includes full-stack app development that interweaves genAI experiences ([e.g.](https://x.com/nickgomezc/status/1770607551379390745?s=20)) and heavy API/SDK/database design.

I'm the cofounder and CEO. We like to iterate quickly, value product and design-oriented engineers, and look for founding engineers to have large scope and ownership from the start.

**Reply by seyedahmaddev** [reply]
hi, do you need remote developer?

**Reply by nick-inkeep** [reply]
Not currently, but thanks for reaching out!

**Reply by apophis51** [reply]
"Inkeep | Full Stack Engineer | In-Person New York or SF | Full-time | 100k-160k base | [Link](https://join.inkeep.com/founding-engineer)"
Link 404 not found at 5:15 a.m EST 4/23/2024
![image](https://github.com/vercel/next.js/assets/49728948/ad3c2d46-f593-4b54-bd8d-7e663405f7b4)

**Comment by timlisiecki** [comment]
Snyk | Web Developer/Engineer | Boston, MA | Full-time | [Link](https://boards.greenhouse.io/snyk/jobs/7315413002)

[Snyk](https://snyk.io/) is looking for a Web Developer, who has hands-on experience in building and optimizing  B2B SaaS websites. In this role, you will work as part of our Web Team within the Global Marketing Team to build a modern, static-generated site using Next.js, and a headless CMS. You will partner with other engineers on the development, optimization, and growth of our marketing websites. Your focus will be full-stack in Next.js as well as development in the headless CMS. Knowledge of SEO and experience building a design system (SCSS library) is a huge plus!

I've been at the company for almost 3 years. I am the Sr. Web Engineer on the team. The culture at Snyk is great, especially on our Web team!

**Reply by pavelbuchta** [reply]
Hi, are you looking for a remote position? If yes, contact me at `pavelbuchta80@gmail.com` and we can discuss further details.

**Comment by Srzzy** [comment]
WTD (Game Company) | Software Engineer (next.js) | Remote (US, Canada, Australia) | Contract | $100-200K / year | https://docs.doiq.xyz/

WTD is an early stage start-up that focuses on building command based games with a community forward approach. At WTD we mix creativity, hardwork, quality and always a pinch of community input to concoct some truly special games. As an early stage start-up, we do things a bit differently. We are a small team who are passionate about creating and pushing the boundaries in new areas of the game-fi and social-fi sectors of the cryptocurrency market. 

You will be responsible for bringing version three of the doiqonomy to life. We are looking for someone who is looking to put full effort behind the initial buildout of this game and feels they have the skills and time to dedicate to the project in these early stages.

**Reply by CantBeSubh** [reply]
hi, I'm intrested. 
Portfolio: https://subhranshu.com/
Email: socials.subhranshu@gmail.com
